monitoring/grafana/dashboards/cephfs-overview.json

```diff
@@ -64,6 +64,7 @@
          "datasource": "$datasource",
          "description": "",
          "fill": 1,
+         "fillGradient": 0,
```

Comment:
@avanthakkar why is this run adding the `fillGradient` and `links: [ ]` to every dashboard? @aaSharma14 ? I guess we pinned the grafonnet version, right? We should ensure this is 'deterministic' and 'reproducible' or we'll be randomly adding and removing fields on every run :S 

In fact this is making `make check` fail...